src/main/kotlin/com/hibob/academy/employeeFeedback/resource/FeedbackResource.kt

```diff
@@ -33,15 +34,60 @@ class FeedbackResource(private val feedbackService: FeedbackService) {
 
     @Path("/feedback")
     @POST
-    fun insertFeedback(feedback: Feedback,@Context requestContext: ContainerRequestContext): Response {
+    fun insertFeedback(feedback: Feedback, @Context requestContext: ContainerRequestContext): Response {
         if (feedback.feedback.length < 10) {
             throw BadRequestException("The feedback is too short.")
         }
         val employeeData = requestContext.getProperty(AuthenticationFilter.EMPLOYEE) as EmployeeData?
             ?: return Response.status(Response.Status.UNAUTHORIZED).build()
         val companyId = employeeData.companyId
-        return Response.ok(feedbackService.insertFeedback(feedback,companyId)).build()
+        return Response.ok(feedbackService.insertFeedback(feedback, companyId)).build()
     }
 
+
+    @Path("/updateFeedbackStatus")
```

Comment:
please define the path based on the resource being accessed, not the action being performed. see [this ](https://docs.google.com/presentation/d/1cPSJ34dhmr-V4crFcRqABLgbwDOA83U1zrSYDlK5osM/edit#slide=id.g889db8d5de_1_13) slidedeck from the REST class